apps/web/pages/apps/[slug].tsx

```diff
@@ -1,12 +1,49 @@
+import fs from "fs";
+import matter from "gray-matter";
 import { GetStaticPaths, GetStaticPathsResult, GetStaticPropsContext } from "next";
+import { MDXRemote } from "next-mdx-remote";
+import { serialize } from "next-mdx-remote/serialize";
+import Image from "next/image";
+import Link from "next/link";
+import path from "path";
 
 import { getAppRegistry } from "@calcom/app-store/_appRegistry";
 
+import useMediaQuery from "@lib/hooks/useMediaQuery";
 import { inferSSRProps } from "@lib/types/inferSSRProps";
 
 import App from "@components/App";
+import Slider from "@components/apps/Slider";
 
-function SingleAppPage({ data }: inferSSRProps<typeof getStaticProps>) {
+const components = {
+  a: ({ href, ...otherProps }) => (
+    <Link href={href}>
+      <a {...otherProps} />
+    </Link>
+  ),
+  img: ({ src, alt = "", ...rest }) => <Image src={src} alt={alt} {...rest} />,
+  Slider: ({ items }) => {
+    const isTabletAndUp = useMediaQuery("(min-width: 960px)");
+    return (
+      <Slider<string>
+        items={items}
+        title="Screenshots"
+        options={{
+          perView: 1,
+        }}
+        renderItem={(item) =>
+          isTabletAndUp ? (
+            <Image src={item} alt="" layout="fixed" width={573} height={382} />
+          ) : (
```

Comment:
There is this weird behaviour with lazily loaded images. Setting loading="eager" fixes the issue.
https://www.loom.com/share/72426c9efe1a4d55bd9c7e69fb60aa4f